plugins/node/instrumentation-undici/src/undici.ts

```diff
@@ -77,26 +76,20 @@ export class UndiciInstrumentation extends InstrumentationBase<UndiciInstrumenta
   }
 
   override disable(): void {
-    if (!this.getConfig().enabled) {
-      return;
-    }
-
     this._channelSubs.forEach(sub => sub.channel.unsubscribe(sub.onMessage));
     this._channelSubs.length = 0;
-    super.disable();
```

Comment:
Note for reviewer: no need to call super since the parent class method deals with patching/unpatching and does not apply here

Reply:
super.disable() is *also* setting `this._enabled = false`, so I wonder if it still should be called.
Those calls to `super.` in the enable() and disable() were added in this PR: https://github.com/open-telemetry/opentelemetry-js-contrib/pull/2289/files#diff-8e6dd9963dd47a60e25c9ac92d7a5ae25ce79c40340cb4721eefd203bec81a60R86

Reply:
I've restored the `super.disable()` and `super.enable()` calls, to allow InstrumentationBase to maintain its internal `this._enabled` setting.